### DISCUSSION TITLE
[Wizard] New maximum height prop for vertical wizard

### DISCUSSION THREAD

**Discussion by marcialfps** [dxc-technology/halstack-react#928] [Polls]
Currently we set the maximum height of the vertical wizard to 500px. With this approach we can run into problems if we have too many steps. The best approach is to provide a new prop that allows users to set the maximum height or a size. 

In this example we have seven steps, but we can only see six.
![image](https://user-images.githubusercontent.com/35959222/164481096-aa54a08d-4c97-4241-8b3f-a22b47398e6e.png)

**Comment by GomezIvann** [comment]
Why should we add a prop for a very specific visual case? I think the wizard should adapt its height to the parent container and have a min-height to avoid steps overlapping.

**Comment by aweell** [comment]
We definitively need to fix the `min-height` issue but I think that setting that value via prop is not the best possible approach. I think this component `height` should be defined only by its content, and if in a particular case the user needs control over it, can be achieved setting a `min-height` or `height` in a wrapper container and control the `y-overflow`...

**Comment by marcialfps** [comment]
[Spark](https://react.sparkdesignsystem.com/?path=/docs/components-stepper--with-step-descriptions), [Ant](https://ant.design/components/steps/) and [Carbon](https://react.carbondesignsystem.com/?path=/story/components-progressindicator--default) which have vertical wizards, don't control the height of the component. I think the best solution is to remove the `max-height` we currently have and allow the user to control the correct behaviour.
